src/momento-redis-adapter.ts

```diff
@@ -181,11 +183,15 @@ export interface MomentoIORedis {
   ): Promise<(string | null)[]>;
 
   flushdb(): Promise<'OK'>;
+
   flushdb(async: 'ASYNC'): Promise<'OK'>;
+
   flushdb(sync: 'SYNC'): Promise<'OK'>;
 
   unlink(...args: [...keys: RedisKey[]]): Promise<number>;
 
+  pipeline(commands?: Array<Array<any>>): ChainableCommander;
```

Comment:
The any here triggers a lot of warnings from GitHub actions. Can look in to suppressing in a follow up PR.

Reply:
Yeah was thinking about that as well.... Had tried `unknown` also but then the caller was seeing issues so switched to any. I agree can keep trying to see if better way to define this in TS in another PR.